llvm/test/Assembler/call-nonzero-program-addrspace-2.ll

```diff
@@ -3,7 +3,7 @@
 
 ; Check that numbered variables in a nonzero program address space 200 can be used in a call instruction
 
-define i8 @test_unnamed(ptr, ptr addrspace(42) %0) {
+define i8 @test_unnamed(ptr, ptr addrspace(42) %1) {
```

Comment:
This is a bug in the previous implementation: While `%0` is written here, it was actually interpreted as `%1`.